### ISSUE TITLE
Inverse fft

### ISSUE THREAD

**Issue by tirthbha** [ubsuny/23-Homework5G4#34]
Are we supposed to perform inverse fft on the original X or the filtered X?

**Comment by sharmistharanit** [comment]
yes we have to take inverse FFT in order to get the plot (raw and cleaned) in time domain. @tirthbha

**Comment by Pranjal-Srivastava-2023** [comment]
> Are we supposed to perform inverse fft on the original X or the filtered X?

@tirthbha yes, we need to take inverse FFT.
